internal/util/logging/logging.go

```diff
@@ -52,10 +52,9 @@ func GoError(err error) slog.Attr {
 	return slog.String("error", fmt.Sprintf("%#v", err))
 }
 
-// Named returns a logger with name segment added to existing name.
-func Named(l *slog.Logger, name string) *slog.Logger {
-	// TODO https://github.com/FerretDB/FerretDB/issues/4431
-	return l
+// FullNamed returns a logger with name attribute set.
+func FullNamed(l *slog.Logger, name string) *slog.Logger {
```

Comment:
What is the difference between "Named" and "FullNamed"?

Reply:
One of them does not exist 😆

Reply:
I mean, why it was important to call the function `FullNamed`, not just `Named`?

Reply:
We discussed that on DS. We might not be able to replicate zap's Named behavior (or the implementation would be very tricky without enough benefits), and having a different name makes sense to avoid usage mistakes

Reply:
Ah, got it now! Thanks!